outputs.tf

```
@@ -92,3 +92,14 @@ output "subnet_share_arn" {
   value       = try(module.subnet_sharing[0].arn, null)
   description = "The ARN of the subnet share in resource access manager if any"
 }
+
+output "vpc_endpoint_ids" {
+  value = {
+    s3          = aws_vpc_endpoint.s3[*].id
```

I think these should all look like this to prevent errors when one isn't created:

```suggestion
    s3          = try(aws_vpc_endpoint.s3[0].id, null)
```

---

I see you updated it, but you updated it using a `*` instead of `0`. But those are two different things and it needs to be a `0` in this case.